### DISCUSSION TITLE
How can I change AppInsightsConnectionString in Settings.xml?

### DISCUSSION THREAD

**Discussion by negberts** [microsoft/service-fabric-observer#290] [Q&A]
I've read the documentation on how to change application parameters from the ApplicationManifest.xml. (https://github.com/microsoft/service-fabric-observer/blob/main/Documentation/Using.md#parameterUpdates) But how can I change the settings for Application Insights or Log Analytics Workspace that are specified in the Settings.xml without having to create a local copy of these sources and changing the values of those?

Is there any way to specify them during deployment or changing them while the application is running on the cluster?

**Comment by negberts** [comment]
@gittorre could you help?

**Reply by GitTorre** [reply]
Telemetry (App Insights, Log Analytics) connection information is not supported as Application Parameters today. Please create a Work Item and we'll address this.
